app/models/github_repo.rb

```diff
@@ -27,9 +27,8 @@ def self.upsert(user, **params)
   end
 
   def self.update_to_latest
-    where("updated_at < ?", 26.hours.ago).ids.each do |repo_id|
-      GithubRepos::RepoSyncWorker.perform_async(repo_id)
-    end
+    ids = where("updated_at < ?", 26.hours.ago).ids.each_slice(1).to_a
```

Comment:
Reading note: `perform_bulk` expects its arguments as an array of arrays, where each inner array contains the arguments for the job it calls. In this case here, we need to provide the repo ids like this: `[[1], [2], ... ]`.